src/page/personal-information.page.ts

```diff
@@ -0,0 +1,87 @@
+import { browser, $, $$, ElementFinder, ElementArrayFinder, by, ExpectedConditions } from 'protractor';
+
+export class PersonalInformationPage {
+  private firstNameInput: ElementFinder;
+  private lastNameInput: ElementFinder;
+  private sexInput: ElementArrayFinder;
+  private experienceInput: ElementArrayFinder;
+  private continentSelector: ElementFinder;
+  private commandsSelector: ElementFinder;
+  private submitButton: ElementFinder;
+
+  constructor() {
+    this.firstNameInput = $('[name="firstname"]');
+    this.lastNameInput = $('[name="lastname"]');
+    this.sexInput = $$('[name="sex"]');
+    this.experienceInput = $$('[name="exp"]');
+    this.continentSelector = $('[name="continents"]');
+    this.commandsSelector = $('[name="selenium_commands"]');
+    this.submitButton = $('[name="submit"]');
```

Comment:
Remember that step 2 of section 19, says that for this case, we should avoid the usage of cssSelectors. So, try to use a different selector :)

Reply:
Done